app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -181,6 +181,7 @@ class ParkingViewModel(
 
   /** set the center of the circle, and reset the radius to DEFAULT_RADIUS */
   fun setCircleCenter(center: Point) {
+    Log.d("ParkingViewModel", "Setting center to $center")
```

Comment:
Forgot to delete it maybe or did you let it on purpose ?

Reply:
Forgot